api/src/main/java/com/codeforcommunity/api/IProtectedReportProcessor.java

```diff
@@ -1,33 +1,40 @@
 package com.codeforcommunity.api;
 
 import com.codeforcommunity.auth.JWTData;
+import com.codeforcommunity.dto.report.GetCommunityStatsResponse;
 import com.codeforcommunity.dto.report.GetReportCSVRequest;
 import com.codeforcommunity.dto.report.GetAdoptionReportResponse;
 import com.codeforcommunity.dto.report.GetStewardshipReportResponse;
 
 public interface IProtectedReportProcessor {
+  /**
+   * Returns the number of current adopters, trees currently adopted,
+   * and all stewardship activities ever performed.
+   */
+  GetCommunityStatsResponse getCommunityStats();
```

Comment:
🔧 Since we want all users to be able to call this, we actually want this to be handled by an `IReportProcessor` instead of `IProtectedReportProcessor`.